src/components/IPScan/Summary/index.js

```diff
@@ -23,7 +23,8 @@ import GoTerms from 'components/GoTerms';
 import Accession from 'components/Accession';
 import ProteinEntryHierarchy from 'components/Protein/ProteinEntryHierarchy';
 import Length from 'components/Protein/Length';
-import { DomainOnProteinWithoutMergedData } from 'components/Related/DomainsOnProtein';
+// $FlowFixMe
```

Comment:
What's the purpose of this comment?

Reply:
The short answer is that is a way to ignore Flow errors.

We are migrating away from FlowJS and implementing all the type validation with TypeScript. But because we are doing it gradually there are files still using Flow that are now importing TS files. Which causes an error in flow, and that's why you see me putting so many of this comments in the migrations PRs